docs/data/material/components/snackbars/snackbars.md

```diff
@@ -9,109 +9,91 @@ waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
 
 # Snackbar
 
-<p class="description">Snackbars provide brief notifications. The component is also known as a toast.</p>
-
-Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn't interrupt the user experience, and they don't require user input to disappear.
-
-Snackbars contain a single line of text directly related to the operation performed.
-They may contain a text action, but no icons. You can use them to display notifications.
+<p class="description">Also known as a toast, snackbars are used for brief notifications of a process that has been performed or will perform.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
+## Basics
```

Comment:
Similar to the comment below — as per the designed Base UI docs structure (https://github.com/mui/material-ui/pull/38529), I guess we'll be changing this very first demo to _Introduction_ instead of either Basics or Simple (just fixed it). cc @samuelsycamore for further guidance!

Reply:
Resolving this one to concentrate the discussion on the comment below!